src/businesslogic/BusinessLogic.java

```diff
@@ -3,10 +3,14 @@
 
 public class BusinessLogic {
 
-
     public static double calculateIva(double price) {
         int iva = 22;
         return (price * iva) / 100;
     }
 
+    public static double getCalculateDiscount(double newPrice, int sconto){
+        double out= (newPrice * sconto)/100;
+        out= newPrice - out;
```

Comment:
perchè newPrice - out ? 
poi non devi passare solo il prezzo nel parametro  ma il prodotto, quindi il prezzo del prodotto senza la sottrazione

Reply:
in piu non si divide per 100....